commit 8fe1b25cb833532135104bb572c48f8f9ec31f9c (refs/changes/75/3975/2)
Author: Pedro Palazon Candel <pedro@joyent.com>
Date:   2018-05-18T18:45:09+02:00 (1 year, 5 months ago)
    
    TRITON-396 TRITON-348 breaks sdcadm call config-agent sync
    Reviewed by: Josh Wilsdon <josh@wilsdon.ca>
    Approved by: Josh Wilsdon <josh@wilsdon.ca>

diff --git a/lib/common.js b/lib/common.js
index 9533f9c..675690a 100644
--- a/lib/common.js
+++ b/lib/common.js
@@ -1114,7 +1114,7 @@ function callConfigAgentSync(opts, cb) {
         opts.server,
         /* JSSTYLED */
         format('/usr/sbin/zlogin %s ' +
-            '"/opt/smartdc/config-agent/build/node/bin/node' +
+            '"/opt/smartdc/config-agent/build/node/bin/node ' +
             '/opt/smartdc/config-agent/agent.js -f ' +
             '/opt/smartdc/config-agent/etc/config.json -s"', opts.vm)
     ];
